File: MK/10월 2주차/숫자의표현.java
class Solution {
    public int solution(int n) {
        int ans=0;

        for (int i=1; i<=n; i++) {
            int val=0;
            for (int j=i; j<=n; j++) {
                val += j;
                if (val==n) {
                    ans++;
                    break;
                }
                if (val > n) {
                    break;
                }
            }
        }
        return ans;
    }
}

File: BW/10월2주차/숫자의표현.js
function solution(n) {
    var startNum = 0;
    var cnt = 0;
    
    for(i=1; i<n; i++){
        startNum=i;
        for(j=i+1; j<n; j++){
            // 현재 값 + 다음 값
            startNum+= j;
            if(startNum === n){
                cnt++;
                break;
            } else if(startNum > n){
                break;
            }
        }
    }
    // n값이 추가되므로 +1
    return cnt+1;
}